bin/citrea/build.rs

```diff
@@ -12,4 +16,47 @@ fn main() {
             .args(["install", "cargo-risczero"])
             .status();
     }
+
+    build_sp1_guest();
+}
+
+fn build_sp1_guest() {
+    println!("cargo:rerun-if-env-changed=BUILD_SP1_GUEST");
```

Comment:
why do we have to explicitly build sp1 guests?

risc0 guests build just fine?

Reply:
this is a choice, not a must. Most likely nothing will not need SP1 guest built, unless we decide to switch to sp1, so I made it opt-in to build instead of opt-out. So we skip the building to not prolong the builds and development even though we don't need it.